recipes/qpdf/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+required_conan_version = ">=1.51.3"
+
+class PackageConan(ConanFile):
+    name = "qpdf"
+    description = "QPDF is a command-line tool and C++ library that performs content-preserving transformations on PDF files."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/qpdf/qpdf"
+    topics = ("pdf")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_crypto": ["native","openssl"],
```

Comment:
```suggestion
        "with_crypto": ["native", "openssl", "gnutls"],
```
Let's add the option now, and keep it as restricted, so as soon as we have that package we just enabled it. The advantage: It should not affect the package Id in the future.